### PR TITLE
Add Teremiare and Innovamania to ssbb + minor fix

### PR DESCRIPTION
**Pull request by Teremiare**

### CODE REVIEW

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -475,6 +474,29 @@ exports.BattleMovedex = {
...
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		stealsBoosts: true,
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, "Spectral Thief", target);
+		},
+		onAfterMoveSecondarySelf: function (source) {
+			this.useMove('batonpass', source);
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js:511]: We shouldnt call other moves unless needed.

Comment (Teremiare) [reply mods/seasonal/moves.js:511]: Joim noted this in the code for ssbm:
> // selfSwitch: 'copyvolatile' does not work with targeted moves.

So calling the move seems to be the only way.

Comment (HoeenCoder) [reply mods/seasonal/moves.js:511]: ok

File: mods/seasonal/statuses.js [hunk 2]
```diff
@@ -60,6 +60,18 @@ exports.BattleStatuses = {
 			this.add('c|%imas|ok');
 		},
 	},
+	innovamania: {
+		exists: true,
+		onStart: function () {
+			sentences = ['Don\'t take this seriously', 'These Black Glasses sure look cool', 'Ready for some fun?( ͡° ͜ʖ ͡°)', '( ͡° ͜ʖ ͡°'];
```
Comment (HoeenCoder) [inline mods/seasonal/statuses.js]: declare this or it will crash

File: mods/seasonal/statuses.js [hunk 3]
```diff
@@ -60,6 +60,18 @@ exports.BattleStatuses = {
...
+			this.add('c|@innovamania|' + sentences[this.random(4)]);
+			this.boost({atk: 6, def: 6, spa: 6, spd: 6, spe: 6, accuracy: 6}, source);
+		},
+		onFaint: function () {
+			sentences = ['Did you rage quit?', 'How\'d you lose with this set?'];
```
Comment (HoeenCoder) [inline mods/seasonal/statuses.js]: and this

**Pull request merged**